### PR TITLE
feat(mainHeader): make the `user menu panel` always open to the left

### PR DESCRIPTION
**Pull request by marcel-bitfly**
See: BEDS-221

### CODE REVIEW

File: frontend/components/bc/header/MainHeader.vue [hunk 1]
```diff
@@ -378,6 +376,13 @@ $smallHeaderThreshold: 1024px;
           }
         }
       }
+      :global(.user-menu-panel) {
+        // hack: panel should always get opened to the left, but this is not possible with component props
+        $widthThePanelHasEnoughSpaceToOpenToTheRight: 1558px;
+        @media screen and (min-width: $widthThePanelHasEnoughSpaceToOpenToTheRight) {
+          translate: -98px;
```
Comment (MauserBitfly) [inline frontend/components/bc/header/MainHeader.vue:383]: Maybe use a more flexible version
```suggestion
          translate: calc(-100% + 44px);
```

if we get items with longer texts in the dropdown
image

**Review by MauserBitfly — COMMENTED** [review]
works fine, I just left a small suggestion

File: frontend/components/bc/header/MainHeader.vue (hunk 1 above)
Comment (marcel-bitfly) [reply frontend/components/bc/header/MainHeader.vue]: very nice, thought about that but gave up to early 💪

**Review by MauserBitfly — APPROVED** [review]
LGTM

**Pull request merged**